examples/speaker_recognition/speaker_reco.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import hydra
+import pytorch_lightning as pl
+
+from nemo.collections.asr.models import EncDecSpeechLabelModel
+from nemo.utils import logging
+from nemo.utils.exp_manager import exp_manager
+
+os.environ["TOKENIZERS_PARALLELISM"] = "false"
+
+"""
+Basic run (on CPU for 50 epochs):
+    python examples/speaker_recognition/speaker_reco.py \
+        model.train_ds.manifest_filepath="<train_manifest_file>" \
+        model.validation_ds.manifest_filepath="<validation_manifest_file>" \
+        hydra.run.dir="." \
```

Comment:
I think we can remove this. If anything we'd probably configure exp_manager.root_dir from CLI

Reply:
Sure